README.md

```diff
@@ -98,23 +99,16 @@ associated with any `opentracing.Span` instance.
 
 #### Deserializing from the wire
 
-```
+```go
     http.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
-        // Grab the TraceContext from the HTTP header using the
-        // opentracing helper.
-        reqTraceCtx, err := opentracing.TraceContextFromHeader(
-                req.Header, opentracing.GlobalTracer())
-        var serverSpan opentracing.Span
-        var goCtx context.Context = ...
+		// Join the trace in the HTTP header using the opentracing helper.
+        serverSpan, err := opentracing.JoinTraceFromHeader(
```

Comment:
important difference: we "JoinTrace..." on the receiving end of a propagated `Span`, much like the previous `JoinTrace` methods do for `Span` instances.

important difference: since there's no `TraceContext` intermediary anymore, we must provide the operation name at "de-propagation" time.